src/communication/azext_communication/generated/_help.py

```diff
@@ -0,0 +1,125 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['communication service'] = """
```

Comment:
It seems that there are only two command groups in `az communication`. If yes, why not move all commands in `service` command group to `communication` level?

Reply:
done